src/core/issues-notes.js

```diff
@@ -14,155 +14,151 @@ import { pub } from "core/pubsubhub";
-            )
-          );
-          $inno.removeAttr("title");
-        }
-        $tit.addClass("marker");
-        $div.append($tit);
-        $inno.replaceWith($div);
-        var body = $inno.removeClass(report.type).removeAttr("data-number");
-        if (ghIssue && !body.text().trim()) {
-          body = ghIssue.body_html;
-        }
-        $div.append(body);
-        const level = $tit.parents("section").length + 2;
-        $tit.attr("aria-level", level);
       }
-      pub(report.type, report);
-    });
-    if ($(".issue").length) {
-      if ($("#issue-summary"))
-        $("#issue-summary").append($issueSummary.contents());
-    } else if ($("#issue-summary").length) {
-      pub("warn", "Using issue summary (#issue-summary) but no issues found.");
-      $("#issue-summary").remove();
+      $tit.find("span").text(text);
+      if (report.title) {
+        $tit.append(
+          $("<span style='text-transform: none'>: " + report.title + "</span>")
+        );
+        $inno.removeAttr("title");
+      }
+      $tit.addClass("marker");
+      $div.append($tit);
+      $inno.replaceWith($div);
+      var body = $inno.removeClass(report.type).removeAttr("data-number");
+      if (ghIssue && !body.text().trim()) {
+        body = ghIssue.body_html;
+      }
+      $div.append(body);
+      const level = $tit.parents("section").length + 2;
+      $tit.attr("aria-level", level);
     }
+    pub(report.type, report);
+  });
+  if ($(".issue").length) {
+    if ($("#issue-summary"))
+      $("#issue-summary").append($issueSummary.contents());
+  } else if ($("#issue-summary").length) {
+    pub("warn", "Using issue summary (#issue-summary) but no issues found.");
+    $("#issue-summary").remove();
   }
-  var $ins = $(".issue, .note, .warning, .ednote"),
-    ghIssues = {},
-    issueBase = conf.issueBase;
-  if ($ins.length) {
-    if (conf.githubAPI) {
-      try {
-        const json = await ghFetch(conf.githubAPI);
-        var issueUrl = issueBase || json.html_url + "/issues/";
-        const issues = await fetchIndex(json.issues_url, {
-          // Get back HTML content instead of markdown
-          // See: https://developer.github.com/v3/media/
-          Accept: "application/vnd.github.v3.html+json",
-        });
-        issues.reduce((issuesObj, issue) => {
-          issuesObj[issue.number] = issue;
-        }, ghIssues);
-      } catch (err) {
-        pub("error", err.message);
-      } finally {
-        handleIssues($ins, ghIssues, issueUrl);
-        cb();
-      }
-  } else {
-      handleIssues($ins, ghIssues, issueBase);
-      cb();
+}
+export async function run(conf) {
+  debugger;
+  const $ins = $(".issue, .note, .warning, .ednote");
+  const ghIssues = {};
+  const { issueBase } = conf;
+  if (!$ins.length) {
+    return; // nothing to do.
+  }
+  if (conf.githubAPI) {
+    try {
+      const json = await ghFetch(conf.githubAPI);
+      const issueUrl = issueBase || json.html_url + "/issues/";
```

Comment:
issueUrl is not being used anywhere, we declared it so that I can pass it to ``` handleIssues ``` function @marcoscaceres